AlphaWallet.xcodeproj/project.pbxproj

```diff
@@ -6029,7 +6029,7 @@
 				87CC3853273E7980005BE9B8 /* eip155URLDecoder.swift in Sources */,
 				5E7C7ECE164289A89734B4EF /* LocalesCoordinator.swift in Sources */,
 				874D2F8027A2B1D1005459DA /* UnstoppableDomainsV2Resolver.swift in Sources */,
-				879F184826E73CF4000602F2 /* Erc1155TokenInstanceViewController.swift in Sources */,
```

Comment:
Looks like the Xcode project file was broken, I have created a separate PR #3872. Can you remove this later?